### ISSUE TITLE
Comment/login problem

### ISSUE THREAD

**Issue by krues8dr** [opengovfoundation/madison#629]
Via Waldo:

> I went to Madison, tried to add a comment to the constitution, and created an account. This took me back to the top. I went down, tried to add a comment, and was prompted to log in. I logged in, which took me back to the top. I scrolled down, tried to add a comment, and I am again prompted to log in. So I’m giving up.

**Comment by krues8dr** [comment]
* [ ] Login returns to home page, not last page visited
* [ ] Successful comment does not reload comments list
* [ ] Can't reproduce the not-logged-in bit, so I'm guessing there's a login error not being shown.

**Comment by waldoj** [comment]
* [ ] creating an account doesn't log in the user

**Comment by krues8dr** [comment]
:+1:

Ok, so it looks like we're not actually letting people login if their account isn't confirmed yet, but it says we're logging them in.  That's a big problem. @cmbirk thoughts?

**Comment by krues8dr** [comment]
I think we can fix the login issue per above, but we need to have a bigger team discussion (and pretty big rearchitecting) about letting users login without being confirmed.

**Comment by waldoj** [comment]
FWIW, I haven't gotten any kind of a confirmation email, nor was I notified at registration time that I would need to look for such an email.

**Comment by krues8dr** [comment]
It claims you did? 
![screen shot 2015-09-17 at 2 35 21 pm](https://cloud.githubusercontent.com/assets/127904/9942578/dd4984bc-5d49-11e5-9ce5-8e105cd6708d.jpg)

**Comment by waldoj** [comment]
Uh. I've got nothin'. I just searched through my mail and I see that I received a confirmation message at 12:26 PM EDT (maybe your system is in CDT?), but it doesn't look at _all_ familiar. My Chrome history has no record of me going to the verification URL, and autocomplete for the URL in the email fails. Maybe I just clicked through that confirmation step so quickly that I immediately forgot all about it? But I don't know why my browser wouldn't have noted it? I dunno.
